llvm/docs/DirectX/DXILOpTableGenDesign.rst

```diff
@@ -172,14 +162,55 @@ Shader Model version is represented as follows:
      int Minor = minor;
    }
 
-   // Valid Shader model version records
+
+Concrete representations of valid Shader Model and DXIL versions are defined as follows:
```

Comment:
One thing that is not clear in the examples below is whether a version represents a specific SM version or a *minimum* version.

It would be good to clarify whether when using `SM6_0` in a constraint means that it is only valid for SM_6_0 or it is valid for SM_6_0+. I guess it also depends on whether there are other shader models in other constraints?

If the meaning is SM_6_0+, then how do we represent something that goes away in a later shader model? For example, `createHandle` is valid up-to SM 6_6, but in SM_6_6 `createHandle` should not appear and should be replaced by `createHandleFromBinding` or `createHandlFromHeap`.